src/leappto/cli.py

```diff
@@ -494,8 +527,8 @@ def _rsync():
                                  .format(rsync_dir, ' '.join(self.target_cfg), self.target_addr)
                     Popen(shlex.split(target_cmd)).wait()
 
-                # temporary, after task with different names for containers should be removed
-                shutil.rmtree(rsync_dir)
+                    # temporary, after task with different names for containers should be removed
+                    shutil.rmtree(rsync_dir)
```

Comment:
@shaded-enmity I agree, after task with unique names for containers ```shutil.rmtree``` should be removed and then we will be able to use power of rsync (I mean "quick check" algorithm in case we want to run migrate-machine again for the same source). Although it's not done yet, that's why I am asking if it's ok to change that line.